kernel/arch/dreamcast/hardware/spu.c

```diff
@@ -210,11 +226,25 @@ void spu_memset_sq(uintptr_t dst, uint32_t what, size_t length) {
     /* Add in the SPU RAM base (cached area) */
     dst |= SPU_RAM_BASE;
 
+    /* First lock SQs because there is a mutex there. */
+    sq_lock(NULL);
```

Comment:
This isn't necessary because `sq_set32()` already locks the SQs?

Reply:
The same.